api/src/main/java/com/mybooks/api/controller/AuthorController.java

```diff
@@ -20,27 +22,28 @@ public AuthorController(AuthorService authorService) {
     }
 
     @GetMapping()
-    List<Author> getAllAuthors() {
-        return authorService.getAllAuthors();
+    ResponseEntity<List<AuthorDTO>> getAllAuthors() {
+        return new ResponseEntity<>(authorService.fetchAll(), HttpStatus.OK);
```

Comment:
Why this changes???
it should be

`@GetMapping()
@ResponseStatus(HttpStatus.OK)
    List<Author> getAllAuthors() {
        return authorService.getAllAuthors();
    }`